Makefile

```diff
@@ -0,0 +1,2 @@
+generate:
```

Comment:
Should we use magefile like the other repos? Will be able to just call the code directly

Reply:
Totally.